types_test.go

```diff
@@ -0,0 +1,43 @@
+package gitlab
+
+import (
+	"testing"
+	"encoding/json"
+)
+
+func TestBoolValue(t *testing.T) {
+	testCases := map[string]struct {
+		data     []byte
+		expected bool
+	}{
+		"should unmarshal true as true": {
+			data:     []byte("true"),
+			expected: true,
+		},
+		"should unmarshal false as true": {
+			data:     []byte("false"),
+			expected: false,
+		},
+		"should unmarshal \"1\" as true": {
+			data: []byte(`"1"`),
+			expected: true,
+		},
+		"should unmarshal \"0\" as false": {
+			data: []byte(`"0"`),
+			expected: false,
+		},
+	}
+
+	for name, testCase := range testCases {
+		t.Run(name, func(t *testing.T) {
+		    var b BoolValue
```

Comment:
Please `go fmt` your code...

Reply:
I don't see what was wrong here. Anyway, I think it is formatted properly now.